src/atomate2_temp/aims/utils/inputs.py

```diff
@@ -12,6 +12,7 @@ class DataFile(MSONable):
 
     objects: Sequence | None = None
 
+    # TARP: Do these classmethods do anything?
```

Comment:
A good question; moreover they seem to work only for plain text files (for binary files they'll throw UnicodeDecodeError). According to the docstring, they are used in cp2k, seems that in our codebase we do not use DataFile classes at all